sormas-backend/src/main/java/de/symeda/sormas/backend/customizableenum/CustomizableEnumFacadeEjb.java

```diff
@@ -204,7 +204,7 @@ public boolean hasEnumValues(CustomizableEnumType type, Disease disease) {
 				.stream()
 				.filter(
 					e -> !disease.isPresent() && CollectionUtils.isEmpty(e.getDiseases())
-							|| e.getDiseases() != null && e.getDiseases().contains(disease.get()))
+							|| disease.isPresent() && e.getDiseases() != null && e.getDiseases().contains(disease.get()))
```

Comment:
Seems to be e bit too much logic for one expression
`if (disease.isPresent())…`?